src/id/comicfx/src/eu/kanade/tachiyomi/extension/id/comicfx/ComicFx.kt

```diff
@@ -207,11 +207,8 @@ class ComicFx : ParsedHttpSource() {
     override fun pageListParse(document: Document): List<Page> {
         val pages = mutableListOf<Page>()
 
-        document.select("#all img").mapIndexed { i, element ->
-            val image = element.attr("abs:src")
-            if (image != "") {
-                pages.add(Page(i, "", image))
-            }
+        Regex(""""page_image":"([^"]*)"""").findAll(document.toString()).asIterable().mapIndexed { i, it ->
+            pages.add(Page(i, "", it.groupValues[1].replace(Regex("""^https?://|/+"""), "")))
         }
```

Comment:
```suggestion
        val pages = document.toString().substringAfter("var pages = ").substringBefore(";")
        return json.parseToJsonElement(pages).jsonArray.mapIndexed { i, it ->
            Page(i, imageUrl = it.jsonObject["page_image"]!!.jsonPrimitive.content
        }
```

Couldn't you parse the `pages` array as a proper JSON array instead of using regex?

Reply:
it works, im just reusing that regex from another ext